tests/unit/source_code/linters/test_python_imports.py

```diff
@@ -27,27 +31,27 @@ def test_linter_returns_list_of_dbutils_notebook_run_calls():
 
 def test_linter_returns_empty_list_of_imports():
     tree = Tree.parse('')
-    assert not ImportSource.extract_from_tree(tree, DependencyProblem)[0]
+    assert not ImportSource.extract_from_tree(tree, DependencyProblem.from_node)[0]
```

Comment:
Why is this required in this PR?

Reply:
It's not required, but as I was touching that file, I noticed the old code was incorrect (it didn't fail because the factory is not used)